docs/source/reference/ipython.rst

```diff
@@ -4,6 +4,7 @@ IPython
 .. image:: ../_static/_images/ipy_logo.png
       :width: 50%
       :target: https://ipython.org
+      :alt: 
```

Comment:
```suggestion
      :alt: IPython Interactive Computing Logo
```

Reply:
> Logos are also often used in the header or banner of a site and are often links which lead to the site’s homepage. These days, many users expect this functionality. At a minimum, the alt text for a company’s logo in the header should be the name of the company. For a slightly better experience, adding additional alt text which says where the link goes can be helpful: for the UserTesting logo, the alt text could read “User Testing – Home”.

[https://www.deque.com/blog/great-alt-text-introduction/](How to Design Great Alt Text: An Introduction)